airrohr-firmware/airrohr-firmware.ino

```diff
@@ -3691,10 +3704,36 @@ String fetchSensorGPSFromAtmega(){
 		}
 	}
 
+	// Obtain GPS send_time from RTC
+	char buf3[40];
+	DateTime now = rtc.now();
+	sprintf(buf3, "%04d-%02d-%02dT%02d:%02d:%02dZ", now.year(), now.month(), now.day(), now.hour(), now.minute(), now.second());
+	timestamp = buf3;
+
+	pcf8575.digitalWrite(P0, HIGH); // turn GPS status led on for 3 seconds
```

Comment:
@Warukira I believe we talked about adding this to a function

Reply:
@chegejames , this function has since been added.